docs/display-settings.md

```diff
@@ -22,7 +22,7 @@ You can also reorder the display of Inserted Ad Positions by placing shortcodes
 
 ## Ad Spacing Using Empty Inserted Ad Positions
 
-You can add Text Widgets to each and every numbered Inserted Ad Position, and they will display every nth paragraph based on the number of paragraphs you defined in **Plugins** > **Ad Inserter**  [options](configuration.md). 
+You can add Text Widgets to each and every numbered Inserted Ad Position, and they will display every nth paragraph based on the number of paragraphs you defined in **Plugins** > **Ad Inserter**  [options](configuration.md).
```

Comment:
Should this say "You can add Widgets" instead of "You can add Text Widgets"? Since you can add other kinds of widgets.